debugging.md

```diff
@@ -617,6 +617,34 @@ time a render box is asked to relayout and repaint. You can use the
 `debugPrintStack()` method from the `services` library to print your
 own stack traces on demand, if this kind of approach is useful to you.
 
+### Measuring app startup time
+
+To gather detailed information about the time it takes for your Flutter app to start, you can run
+the `flutter run` command with the `trace-startup` option.
+
+```
+$ flutter run --trace-startup
```

Comment:
Also suggest to pass `--profile`. Otherwise, we will run in `--debug` mode, which is slow (on purpose) and will not give you a real number.